keras/layers/layer_test.py

```diff
@@ -632,6 +633,7 @@ def call(self, x):
             layer1.non_trainable_variables, non_trainable_variables
         ):
             self.assertAllClose(ref_v, v)
+        self.assertLen(losses, 2)
         for ref_loss, loss in zip(layer1.losses, losses):
```

Comment:
So previously the test worked because the zip was zipping lists of different lengths?

Reply:
I think previously there was only a single loss there. If you look above at the same file, I added a regularization loss (so now it's 2 in total).